### DISCUSSION TITLE
Cant join the server

### DISCUSSION THREAD

**Discussion by Jaceideu** [MCPHackers/RetroMCP-Java#144] [Q&A] [answered]
When I'm launching server from MCP, I can't join it, the game shows "Logging in" idefinietly. I tried it on modified beta 1.7.3. But it also happens on fresh beta 1.7.3.

**Comment by mineLdiver** [comment, marked as answer]
Have you turned off "online-mode" in server.properties?

**Reply by Jaceideu** [reply]
Thank you, couldn't find those setting at first, that was it.
